internal/template/template_test.go

```diff
@@ -91,6 +92,40 @@ func TestMergeData(t *testing.T) {
 	}
 }
 
+func TestCopyData(t *testing.T) {
+	nested := map[string]interface{}{
+		"c": []int{1, 2, 3},
+	}
+	addrNested := fmt.Sprintf("%p", nested)
```

Comment:
`addrOrginal` might be a more intuitive var name

Reply:
I removed comparing addresses from the test.